roles/internal/Islandora-Devops.alpaca/defaults/main.yml

```diff
@@ -3,80 +3,71 @@
 alpaca_from_source: no
 alpaca_version: main
 alpaca_clone_directory: /opt/alpaca
+alpaca_jar_path: "{{ alpaca_clone_directory }}/islandora-alpaca-app/build/libs/islandora-alpaca-app-2.0.0-all.jar"
 
-alpaca_karaf_repos:
-  - mvn:org.apache.camel.karaf/apache-camel/2.20.4/xml/features
-  - mvn:org.apache.activemq/activemq-karaf/5.15.0/xml/features
-  - mvn:ca.islandora.alpaca/islandora-karaf/1.0.1/xml/features
+alpaca_user: vagrant
```

Comment:
This should be the ansible default ssh user variable and not 'vagrant' as this may not be being run on Vagrant.

Reply:
So I changed this to
```
alpaca_user: "{{ ansible_user }}"
alpaca_group: "{{ ansible_user }}"
```
now I get this

```
TASK [Islandora-Devops.alpaca : Install Alpaca via Gradle] *********************
Monday 20 December 2021  11:39:17 -0600 (0:00:00.316)       0:58:00.653 ******* 
fatal: [default]: FAILED! => {"msg": "The field 'become_user' has an invalid value, which includes an undefined variable. The error was: {{ ansible_user }}: 'ansible_user' is undefined. {{ ansible_user }}: 'ansible_user' is undefined"}
```

🤷 
p.s. I used `ansible_user` and not `ansible_ssh_user` because of the note here https://docs.ansible.com/ansible/2.3/intro_inventory.html#list-of-behavioral-inventory-parameters

Reply:
I also tried `ansible_ssh_user` and it also fails... :shrug:

Reply:
@alxp I'm going to leave this as is because I can't figure out what is supposed to work. You can override it with your own ansible config and set it to whatever you need.

Reply:
Actually for some reason it is using `ansible_user` in `become_user` that is the problem. Otherwise it is valid, but there it causes a failure. I'll do a little more digging